advisor/src/main/kotlin/Advisor.kt

```diff
@@ -45,7 +45,7 @@ import org.ossreviewtoolkit.utils.showStackTrace
  */
 abstract class Advisor(val advisorName: String, protected val config: AdvisorConfiguration) {
     companion object {
-        private val LOADER = ServiceLoader.load(AdvisorFactory::class.java)!!
+        private val LOADER = ServiceLoader.load(VulnerabilityProviderFactory::class.java)!!
```

Comment:
Commit message:

> Prepare the name

I do not get what that part is supposed to mean, sorry. Can you maybe rephrase?

> as the advisors will inherit from the VulnerabilityProvider class in the following commit.

In the next commit the abstract `VulnerabilityProvider` class is only introduced, not used. So nothing is inherited. Also, I'm confused about the order of inheritance described here: An `Advisor` (as a concept / base class) is more generic than a `VulnerabilityProvider`, so if at all, `VulnerabilityProvider` should inherit from `Advisor`.

I guess what you intended to say is that the current advisor *implementations* will start to be implemented as `VulnerabilityProvider` instead. Right?

Reply:
Yes, I meant the latter one. I will rephrase the commit message.